lib/utils.js

```diff
@@ -105,3 +105,16 @@ exports.roughSizeOfObject = function (object) {
 exports.buildCollectionURL = function (base, dbName, collectionName) {
   return base + 'db/' + dbName + '/' + encodeURIComponent(collectionName);
 };
+
+exports.isValidDatabaseName = function (name) {
+  if (!name || name.length === 0 || name.length > 63) {
```

Comment:
Technically, you don't really need the name.length === 0 check, as an empty string is falsy.